src/components/popup/Token.tsx

```diff
@@ -248,50 +228,6 @@ export const LogoAndDetails = styled.div`
   gap: 0.8rem;
 `;
 
-// TODO: this could be removed to the components it's being used in
-export const LogoWrapper = styled(Squircle)<{ small?: boolean }>`
-  position: relative;
-  width: ${(props) => (props.small ? "2.1875rem" : "2.8rem;")};
-  height: ${(props) => (props.small ? "2.1875rem" : "2.8rem;")};
-  flex-shrink: 0;
-  color: rgba(${(props) => props.theme.theme}, 0.2);
-  display: flex;
-  align-items: center;
-  justify-content: center;
-`;
-
-export const Logo = styled(Image).attrs({
-  draggable: false,
-})<{ width?: number; height?: number }>`
-  position: relative;
-  flex-shrink: 0;
-  border-radius: 29px;
-  object-fit: cover;
-  overflow: hidden;
-  z-index: 1;
-  width: ${(props) => (props.width ? `${props.width}px` : "40px")};
-  height: ${(props) => (props.height ? `${props.height}px` : "40px")};
-
-  &::before {
```

Comment:
The background fill is no longer needed because the AR logo properly changes to dark or light depending on the theme.

For other tokens, I'd argue they should not be expecting their provided image to be filled in with background colors, so it should be good too.